{"uuid": "8f60cead-a465-4f22-8950-3096dd025904", "children": ["227bea4d-b488-46cb-a5f6-42cba7927352"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.NoSuchDriverException: Message: Unable to obtain driver for chrome; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors/driver_location\n\n", "trace": "  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 53, in run_old_style_hookwrapper\n    return result.get_result()\n           ~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\pluggy\\_result.py\", line 103, in get_result\n    raise exc.with_traceback(tb)\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\tests\\conftest.py\", line 36, in browser\n    driver = webdriver.Chrome(service=service, options=options)\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 47, in __init__\n    super().__init__(\n    ~~~~~~~~~~~~~~~~^\n        browser_name=DesiredCapabilities.CHROME[\"browserName\"],\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    ...<3 lines>...\n        keep_alive=keep_alive,\n        ^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 53, in __init__\n    if finder.get_browser_path():\n       ~~~~~~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py\", line 47, in get_browser_path\n    return self._binary_paths()[\"browser_path\"]\n           ~~~~~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\timon\\Documents\\Testing\\Automation\\automation_epale\\venv\\Lib\\site-packages\\selenium\\webdriver\\common\\driver_finder.py\", line 78, in _binary_paths\n    raise NoSuchDriverException(msg) from err\n"}, "start": 1751524427287, "stop": 1751524427287}], "start": 1751524427287, "stop": 1751524427463}